<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>56</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>size</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>transparency</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0 px</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0 px</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>251</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>0</number>
       </property>
       <widget class="QPushButton" name="pushButton_30">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/attribution-pencil.svg</normaloff>icons/attribution-pencil.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_21">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/circle.svg</normaloff>icons/circle.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_27">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/stop.svg</normaloff>icons/stop.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="colour_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/algorithm.svg</normaloff>icons/algorithm.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="colour_3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/palette.svg</normaloff>icons/palette.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>251</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>251</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>0</number>
       </property>
       <widget class="QPushButton" name="eraser">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/eraser.svg</normaloff>icons/eraser.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="fill">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/fill.svg</normaloff>icons/fill.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="tap">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/tap.svg</normaloff>icons/tap.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="colour_5">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/palette.svg</normaloff>icons/palette.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="colour_7">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/palette.svg</normaloff>icons/palette.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
